search-resources-api/src/main/java/no/unit/nva/search/SearchAllHandler.java

```diff
@@ -0,0 +1,137 @@
+package no.unit.nva.search;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.util.function.Predicate.isEqual;
+import static no.unit.nva.search.SearchClientConfig.defaultSearchClient;
+import static no.unit.nva.search.constants.ApplicationConstants.objectMapperWithEmpty;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.URI;
+import java.util.Optional;
+import java.util.Set;
+import no.unit.nva.search.models.SearchResourcesResponse;
+import no.unit.nva.search.restclients.IdentityClient;
+import no.unit.nva.search.restclients.IdentityClientImpl;
+import no.unit.nva.search.restclients.responses.UserResponse;
+import no.unit.nva.search.restclients.responses.ViewingScope;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.ForbiddenException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.attempt.Try;
+import nva.commons.core.paths.UriWrapper;
+import org.elasticsearch.action.search.SearchResponse;
+
+
+public class SearchAllHandler extends ApiGatewayHandler<Void, SearchResourcesResponse> {
```

Comment:
A copy past of the SearchHandler, the only difference is that we send both index names as parameters to the query